warehouse/api/schema.py

```diff
@@ -0,0 +1,195 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+
+from marshmallow import Schema, fields
+
+
+class File(Schema):
+    filename = fields.Str()
+    packagetype = fields.Str()
+    python_version = fields.Str()
+    has_sig = fields.Bool(attribute="has_signature")
```

Comment:
Why change the name?

Reply:
`has_sig` is what the JSON API uses, so I started there. I prefer to keep the names consistent with the db where possible, I'll change it to has_signature.